lib/improver/tests/cube_combiner/test_CubeCombiner.py

```diff
@@ -70,6 +73,69 @@ def test_basic(self):
         self.assertEqual(result, msg)
 
 
+class Test_expand_bounds(IrisTest):
+
+    """Test expand_bounds method"""
+
+    def setUp(self):
+        """Set up a cubelist for testing"""
+        list_of_cubes = []
+        for time in range(402193, 402196):
+            cube = set_up_percentile_temperature_cube()[0]
```

Comment:
You are not using the percentile part so not sure why you are using set_up_percentile_temperature_cube to create the cube. However there is nothing really wrong with this as it is only sample data. At some point we need a generic cube creator for our unit tests so we can get rid of this proliferation of cube generators.